templates/fragments/_rootcheck.erb

```diff
@@ -35,6 +35,11 @@
   <% if @ossec_rootcheck_rootkit_trojans-%>
   <rootkit_trojans><%= @ossec_rootcheck_rootkit_trojans %></rootkit_trojans>
   <%- end -%>
+  <%- if @ossec_rootcheck_system_audit -%>
```

Comment:
I would check if the list is empty or not, rather than checking if it's defined.

Changed in 96e2e8993db1c4a391228d69c181e756ab8cb045

Thanks,

Reply:
@rshad ah! my bad, thanks for pointing it out and fixing it :)